Dockerfile

```diff
@@ -10,6 +10,7 @@ WORKDIR /nodeApp
 ADD . /nodeApp
 RUN ./make_version.sh
 
+# why is this not "npm install --production"? I'd like to exchange it with "npm prune --production"
```

Comment:
actually, I just tested my own suggestion and it breaks things in that you can't `npm run install` after pruning, because napa ( a dev dependency ) is itself removed. Pruning also removes the (virtual, unknown module `ckeditor-lineheight-plugin` that napa installs. This is too complicated just for a ckeditor plugin!

Anyway, I think the correct way might be to add `npm run install` **after** `npm update --dev`, and then replace these two lines with:

```
npm prune --production
npm install napa
npm run install
```
It should work for now, but maybe we should rethink about how we import this one ckeditor plugin